### PR TITLE
dev-feature-3

### PR DESCRIPTION
**Pull request by tulenh307**
dev-feature-3 : tulenh308 help review

### CODE REVIEW

File: ProjectServiceCustom.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+    private static final String GITLAB_PROJECT_URL = "https://gitlab.com/api/v4/projects?membership=true&access_token=";
+
+    @Override
+    public List<Project> getAllGitLabProjects(String token) {
+        String url = GITLAB_PROJECT_URL.concat(token);
+        ResponseEntity<GitLabProjectWrapper[]> responses = httpService.get(url, null, GitLabProjectWrapper[].class);
+        List<Project> projects = new ArrayList<>();
+
+        for (GitLabProjectWrapper response : responses.getBody()) {
```
Comment (tulenh308) [inline ProjectServiceCustom.java:29]: NOT OK : var response not use dev-feature-3

File: ProjectServiceCustom.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            Project project = new Project();
+            project.setId(response.getId());
+            project.setName(response.getName());
+            projects.add(project);
+        }
+
```
Comment (tulenh308) [inline ProjectServiceCustom.java:35]: NOT OK : not use connect database in loop dev-feature-3

**Review by tulenh308 — COMMENTED** [review]
NOT OK : tulenh307 recheck

**Review by chungdt2019 — APPROVED** [review]
OK 2019/07/03-5 : Approve

**Pull request merged**